### PR TITLE
changes to startup and sample app doc as part of review when attempti…

### PR DESCRIPTION
**Pull request by gabemontero**
…ng to repro issue 2740

@bparees 
Ben - here is my take on the doc updates we discussed via email.  In case this context helps, I went back and forth on some form of update that warns the user about HW/OS enablement for virtualization.  Hopefully you agree that the RHEL link I provided is in line with the flavor of hint you were comfortable with.

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -15,10 +15,10 @@ Or if you are interested in development, start with:
 
 * <<openshift-development>>
 
-And choose between:
+	And choose between:
```
Comment (bparees) [inline CONTRIBUTING.adoc]: this indent is making markdown interpret it as a code/preformatted blog.  i'm not a markdown guru but you'll need to find another way to do this.

File: examples/sample-app/README.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ This will set up a go workspace locally and will build all go components.  It is
 
 ### Docker Changes ###
 
+**VAGRANT USERS**:
+If you are using a Vagrant image you can skip this step.
```
Comment (bparees) [inline examples/sample-app/README.md]: s/a/the/ since it's not true for just any vagrant image, only ours.

File: examples/sample-app/README.md [hunk 3]
```diff
@@ -68,6 +71,9 @@ These instructions assume you have not changed the kubernetes/openshift service
 
 ### FirewallD Changes ###
 
+**VAGRANT USERS**:
+If you are using a Vagrant image you can skip this step.
```
Comment (bparees) [inline examples/sample-app/README.md]: s/a/the/ since it's not true for just any vagrant image, only ours.

File: examples/sample-app/README.md [hunk 4]
```diff
@@ -163,26 +169,17 @@ This section covers how to perform all the steps of building, deploying, and upd
...
-
-        $ curl -H 'Accept: application/json' -v `oc get service docker-registry --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}/v2/" --config=openshift.local.config/master/admin.kubeconfig`
-
-    You should see:
-
-        "< Docker-Distribution-Api-Version: registry/2.0"
-
-
-7. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
+6. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
```
Comment (bparees) [inline examples/sample-app/README.md]: I think this is unnecessary now too.  later in the steps they create a non-default project and create their app there, so there's no need to grant them access to the default project.

Note that until that project is created, they won't be able to see anything in the web console if they login as test-admin.

File: examples/sample-app/README.md [hunk 5]
```diff
@@ -163,26 +169,17 @@ This section covers how to perform all the steps of building, deploying, and upd
...
 
 
-9. Login as `test-admin` using any password
+7. Login as `test-admin` using any password
 
         $ oc login --certificate-authority=openshift.local.config/master/ca.crt
 
 
-9. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8443/console`.  Login using the user `test-admin` and any password.
+8. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8443/console`.  Login using the user `test-admin` and any password.
```
Comment (bparees) [inline examples/sample-app/README.md]: Delete this step.  somehow we ended up with two of them?   (this one and step 10).  Anyway it makes more sense to have them login to the console after creating the project in step 9. that way they will actually be able to see something when they login.

**Comment by bparees** [comment]
@gabemontero the link looks fine, couple other comments on the steps though.

**Comment by gabemontero** [comment]
OK, I've incorporated the responses, as well as went back and added the new oc command for processing the json file that Cesar pointed us to yesterday.

**Comment by bparees** [comment]
@gabemontero have you walked through this finalized set of steps to confirm it all works as written?

**Comment by gabemontero** [comment]
I've done the non-optional ones.  I'll get started on the optional/advanced ones, but tell me if that is overkill.

**Comment by bparees** [comment]
yeah that's probably overkill.

@smarterclayton @danmcp this is a docs only change and it's a pretty useful one, can we get it approved please?

**Comment by smarterclayton** [comment]
Approved

**Comment by bparees** [comment]
[merge]

**Comment by gabemontero** [comment]
@bparees OK Ben, I've pushed the changes from the Advanced section walkthrough.  The "fixes stemming from advanced section walkthrough"

**Comment by bparees** [comment]
one minor issue and needs a squash

**Comment by bparees** [comment]
@gabemontero you've got a merge conflict now (evidently someone else changed this file while you were working on it).  i'll show you how to deal w/ that tomorrow.

**Comment by gabemontero** [comment]
OK thanks.  Yep, also got a message from the Openshift bot saying a rebase had to occur in order to merge.

----- Original Message -----

**Comment by bparees** [comment]
[merge]

**Pull request merged**